proto/asn1/asn1-universal-types/asn1_universal_types.proto

```diff
@@ -0,0 +1,77 @@
+// This proto represents the Universal types
+// found in X509 Certifiactes.
+syntax = "proto2";
+
+package asn1_universal_types;
+
+// See X.690 (2015), 8.1.2.5.
+enum Encoding {
+  Primitive = 0;
+  Constructed = 1;
+}
+
+// See X.690 (2015), 8.6.
+message BitString {
+  // BitString is either constructed or primitive (X.690 (2015), 8.6.1).
+  required Encoding encoding = 1;
+  required bytes val = 2;
+}
+
+// See X.690 (2015), 8.3.
+message Integer {
+  required bytes val = 2;
+}
+
+// See X.690 (2015), 11.8.
```

Comment:
Similar to the comment above, I think we want to expand on this design a bit, and why this approach was used for both of these types.

There are a number of ways we could represent this, and so this is where a comment is useful to explain why, for example, we didn't use [`google.protobuf.Timestamp`](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Timestamp) to represent this.

Reply:
I ended up using `google.protobuf.Timestamp`, thank you for the suggestion.